.github/workflows/cabal.yml

```diff
@@ -82,3 +75,7 @@ jobs:
       with:
         name: ${{ matrix.os }}-exe
         path: bin
+
+  check:
```

Comment:
What does this do?

Reply:
Nothing.  But we can move the required checks to this job instead of using required checks on things that change regularly:

![image](https://user-images.githubusercontent.com/63014/114128109-fdcc9f80-993e-11eb-85fb-03d3e339f6c6.png)

Reply:
Oh I see - could you add a comment to the yaml about that?

Reply:
Done!